client/src/components/pages/RobotOverview/RobotOverview.jsx

```diff
@@ -13,56 +17,101 @@ const { Search } = Input;
+  const [userId, setUserId] = useState(1);
+
+  /**
+   * @description Equivalent to ComponentDidMount in class based components
+   */
+  useEffect(() => {
+    getCurrentUserId().then((data) => {
+      setUserId(data.UserId);
+    });
+  }, []);
+
+  /**
+   * @description Updates the current user id on the server side and set the state variable userId to the new value
+   */
+  const changeUserId = (value) => {
+    setCurrentUserId(value).then((data) => {
+      setUserId(data.UserId);
+    });
+  };
 
   const updateSearchValue = (val) => {
     setSearchValue(val);
   };
 
   const createRobotBoxes = (searchValue2) => {
     // mock object (JSON of example-robots)
-    const robotList = [{
-      "robotMetadata": { "robotId": "#1234", "robotName": "EXCEL Workflow" }
-    },
-    {
-      "robotMetadata": { "robotId": "#1234", "robotName": "alle E-Mails löschen" }
-    },
-    {
-      "robotMetadata": { "robotId": "#1234", "robotName": "Daily CheckIn" }
-    },
-    {
-      "robotMetadata": { "robotId": "#1234", "robotName": "Daily Checkout" }
-    },
-    {
-      "robotMetadata": { "robotId": "#1234", "robotName": "Twitter checken" }
-    },
-    {
-      "robotMetadata": { "robotId": "#1234", "robotName": "Login to PayPal" }
-    },
+    const robotList = [
+      {
+        robotMetadata: { robotId: '#1234', robotName: 'EXCEL Workflow' },
+      },
+      {
+        robotMetadata: { robotId: '#1234', robotName: 'alle E-Mails löschen' },
+      },
+      {
+        robotMetadata: { robotId: '#1234', robotName: 'Daily CheckIn' },
+      },
+      {
+        robotMetadata: { robotId: '#1234', robotName: 'Daily Checkout' },
+      },
+      {
+        robotMetadata: { robotId: '#1234', robotName: 'Twitter checken' },
+      },
+      {
+        robotMetadata: { robotId: '#1234', robotName: 'Login to PayPal' },
+      },
     ];
 
     const filteredBotList = Object.values(robotList).filter((val) =>
-      val.robotMetadata.robotName.toUpperCase().includes(searchValue2.toUpperCase()));
+      val.robotMetadata.robotName
+        .toUpperCase()
+        .includes(searchValue2.toUpperCase())
+    );
 
-    return <>
-      {filteredBotList.map((val) => (
-        <RobotContainer robotName={val.robotMetadata.robotName} />
-      ))}
-    </>
+    return (
+      <>
+        {filteredBotList.map((val) => (
+          <RobotContainer robotName={val.robotMetadata.robotName} />
+        ))}
+      </>
+    );
   };
 
   return (
     <Layout>
       <HeaderNavbar selectedKey={1} />
 
-      <Space style={{ padding: "1rem" }} direction='vertical' size='middle' >
-        <Search placeholder='Search your Robot!' onSearch={updateSearchValue} enterButton />
+      <Space style={{ padding: '1rem' }} direction='vertical' size='middle'>
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'row',
+          }}
+        >
+          <Search
+            style={{
+              marginRight: '16px',
```

Comment:
Please use 1rem instead of 16px (then we have the exact same margin between all components)